rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/validator/CovidImageViewFactory.java

```diff
@@ -16,6 +17,17 @@
 
     @Override
     protected Bitmap getBitmap(WidgetArgs widgetArgs) {
-        return RDTJsonFormUtils.convertBase64StrToBitmap(widgetArgs.getJsonObject().optString(BASE64_ENCODED_IMG));
+        final int degrees = 90;
```

Comment:
hmm, this is fine, only noting that for an arg only used in one place, I'd expect it outside this function as a constant, or passed as a function parameter (but I guess this option isn't relevant in Java since it doesn't support default arguments/parameters)

Reply:
Yea, this is more of a Codacy thing where it complains about magic numbers a lot.

Reply:
cool, yea and this serves the goal of documenting what that number means and it's units so that's nice, people know not to write pi/2 :)